include/rocksdb/options.h

```diff
@@ -1600,6 +1600,21 @@ struct ReadOptions {
   // Default: false
   bool adaptive_readahead;
 
+  // For file reads associated with this operation, charge the internal rate
```

Comment:
"..with this option.." instead of operation?

Reply:
Done.